postman/Configurations.postman_collection.json

```diff
@@ -18,7 +18,7 @@
 				],
 				"body": {
 					"mode": "raw",
-					"raw": "{\n    \"id\": \"\",  \r\n    \"projectFileConfigurations\": [\r\n        {\"path\": \"\", \"inputs\": [\r\n            {\"name\": \"\", \"typedValue\": {\"_type\": \"\", \"value\": \"\"}},\r\n            {\"name\": \"\", \"typedValue\": {\"_type\": \"\", \"value\": \"\"}}\r\n            ]}\r\n    ]\r\n}",
+					"raw": "{\n    \"id\": \"\",  \r\n    \"wdlParams\": [\r\n        {\"path\": \"\", \"inputs\": [\r\n            {\"name\": \"\", \"typedValue\": {\"_type\": \"\", \"value\": \"\"}},\r\n            {\"name\": \"\", \"typedValue\": {\"_type\": \"\", \"value\": \"\"}}\r\n            ]}\r\n    ]\r\n}",
```

Comment:
I don't think that you've made enough changes. Try to upload configuration using this example and see what happens

Reply:
Fixed: moved `Build project configuration by project` from files to configurations in postman collections. Path edited

Reply:
Nice! But I wasn't talking about moving another endpoint to this collection (although it is definetely needed too). This particular line still contains mistake.

Reply:
Add configuration request body (raw) is now updated to correspond to changed `ProjectConfiguration`structure (only one wdlParams instead of seq). Raw body changed a little to look pretty in postman.
Hope i got it at last :)